app/src/utils/api.ts

```diff
@@ -0,0 +1,66 @@
+import data from "../../src/data/lanes";
+import { getItemsCountFromUUIDs } from "../../src/utils/utils";
+import type { LaneDataType } from "../../src/types/Lane";
+import { ENV_KEY } from "../../src/types/EnvironmentType";
+import {
+  getNumDigitizedItems,
+  getFeaturedImage,
+  imageURL,
+} from "../utils/utils";
+import appConfig from "../appConfig";
+
+export const getHomePageData = async () => {
```

Comment:
Add a ticket in the backlog to write tests for `getHomePageData` and `getFeaturedItemData`. Thanks!

Reply:
https://newyorkpubliclibrary.atlassian.net/browse/DR-3170